src/index.ts

```diff
@@ -290,19 +292,37 @@ export class JolocomSDK {
     await this.storage.store.identity(id)
   }
 
-  public async deleteIdentityData(did: string): Promise<void> {
-    let identity = await this.resolve(did)
-    await this.storage.delete.encryptedWallet(did)
-    await this.storage.delete.verifiableCredentials([
-      { subject: did },
-      { issuer: did },
-    ])
-    await this.storage.delete.identity(did)
-    let query: InteractionQueryAttrs[] = []
-    identity.publicKeySection.forEach((pk) => {
-      let keyId = `${did}${pk.id}`
-      query.push({ initiator: keyId }, { responder: keyId })
-    })
-    await this.storage.delete.interactions(query)
+  /**
+   * Deletes data associated with an identity
+   *
+   * @param did - Identity's DID
+   * @param options - Delete options
+   */
+  public async deleteIdentityData(
```

Comment:
Could you also do me a favor and rename/refactor to make it `sdk.deleteAgent()` :smile: 
I made a mistake naming this similar to `storeIdentityData` but now I think it's clearly a sibling of `initAgent`, `loadAgent`, etc
Please also remember to rename `DeleteIdentityDataOption` etc